src/condor_tools/htcondor_cli/eventlog.py

```diff
@@ -0,0 +1,117 @@
+import os
+import re
+
+from pathlib import Path
+
+import htcondor
+
+from htcondor_cli.noun import Noun
+from htcondor_cli.verb import Verb
+from datetime import datetime
+from datetime import timedelta
+
+class Read(Verb):
+    """
+    Reads an eventlog
+    """
+
+    options = {
+        "log_file": {
+            "args": ("log_file",),
+            "help": "Log file",
+        }
+    }
+
+    def __init__(self, logger, log_file, **options):
+        # make sure the specified log file exists and is readable
+        log_file = Path(log_file)
+        if not log_file.exists():
+            raise FileNotFoundError(f"Could not find file: {str(log_file)}")
+        if os.access(log_file, os.R_OK) is False:
+            raise PermissionError(f"Could not access file: {str(log_file)}")
+        
+        # dictionary to store summary of events for each job
+        event_summaries = {}
+
+        with htcondor.JobEventLog(str(log_file)) as event_log:
+            for event in event_log.events(0):
```

Comment:
Consider
```suggestion
            for event in event_log.events(wait_for=0):
```
so that people who aren't familair with the `JobEventLog` class can read this code more easily.